crates/common/tedge_config/src/sudo.rs

```diff
@@ -65,4 +66,74 @@ impl SudoCommandBuilder {
             }
         }
     }
+
+    /// Ensure the command can be executed using sudo
+    ///
+    /// Be warned, that the command is actually executed.
+    pub fn ensure_command_succeeds<S: AsRef<OsStr>>(
+        &self,
+        program: &impl AsRef<OsStr>,
+        args: &Vec<S>,
+    ) -> Result<(), SudoError> {
+        let status = self
+            .command(program)
+            .args(args)
+            .stdout(Stdio::null())
+            .stderr(Stdio::null())
```

Comment:
It might be a misunderstanding on my side, but I find redirecting stdout/stderr to dev null surprising given the the stdout and stderr maybe part of the plugin contract (e.g. the log plugin uses stdout to get the list of supported log types from a plugin).

Reply:
I believe the intention of the method is that we're just verifying that the plugin obeys this bit of the contract (i.e. it's executable and in that specific case, supports a list command) rather than anything where we actually particularly care about the output.